### ISSUE TITLE
Scripts

### ISSUE THREAD

**Issue by chadob** [AlegriaForman/portfolio#12]
The about.js, contact.js, and skills.json can all probably be combined into a single json file. That'll help a little bit with keeping everything readable.

Portfolio.js: 
Line 10-41: you could probably reduce this all to like 5-10 lines of code. We did this in an example in class where we hid all elements of a class and then showed one of those elements that had a different class based on what was clicked.

Speaking of classes...You have a bunch of id's in the index.html page. We talked about saving id's for javascript and using classes for css selectors. You could switch those id's to classes.

PortfolioFunc.js: 
It was recommended to wrap everything in an iife and export it. You could look at what we did in past assignments and replicate that. 
Line 10-39 could probably be reduced to one template call with a forEach function and by moving everything to one json file.
Some of the naming is a little difficult to follow. You have a fetchSkillsPics, loadSkillsPics, and getSkillsPics. It's a little hard to decipher what does what.

Line 62-76 can probably be reduced to two methods by being less specific with the array that is pushed to and element that is appended. These could be added as parameters

**Comment by AlegriaForman** [comment]
Agreed.  I know there's a better approach, which according to Brian, we will cover on Monday.  I will refactor about.js, contact.js and skills.json when cover the subject matter.

Portfolio.js
Lines 14, 24 and 34 are duplicates and were removed.  The rest of the ids may look the same but are in fact unique elements that I wanted to hide or show whenever a click event is handled.

PortfolioFunc.js: 
Agreed, placed the entire code in an IFFE.  Lines 10 thru 39 will be refactored as soon as we cover the subject on Monday.  The naming conventions were based on the previous assignments i.e. fetch\* | load\* | get*. Lines 62-76 will also be refactored as soon as we cover client side routing.

**Comment by AlegriaForman** [comment]
Corrections:
Lines 14, 24 and 34 were not duplicates.  I did however group them with the other elements that are supposed be hidden.
